### ISSUE TITLE
Error: spawn cmd.exe ENOENT

### ISSUE THREAD

**Issue by jan-dolejsi** [adam8810/vscode-git-rename#11]
For a while now (perhaps a year?) the extension silently fails to execute the `git mv`. The extension still shows the prompt for the new name and then nothing happens, when you accept the new name. 

I finally looked in the logs and found this:

```
[Extension Host] git mv "c:\origpath" "c:\newpath"
log.ts:313   ERR spawn cmd.exe ENOENT: Error: spawn cmd.exe ENOENT
	at Process.ChildProcess._handle.onexit (node:internal/child_process:282:19)
	at onErrorNT (node:internal/child_process:477:16)
	at processTicksAndRejections (node:internal/process/task_queues:83:21)
```

I am running on Windows 10 and I tried to select several different default terminals, in case it mattered. But it just seems to call the node spawn.

**Comment by sgbaird** [comment]
I'm seeing the same behavior where it silently fails https://www.screencast.com/t/hIkbvHyzhjH
```python
[2022-06-17 11:07:00.919] [renderer1] [error] spawn cmd.exe ENOENT: Error: spawn cmd.exe ENOENT
	at Process.ChildProcess._handle.onexit (node:internal/child_process:282:19)
	at onErrorNT (node:internal/child_process:477:16)
	at processTicksAndRejections (node:internal/process/task_queues:83:21)
```

**Comment by tjamecny** [comment]
Same here (Win10, VScode 1.68.1, git-rename 1.2.0):
```
[2022-06-20 15:23:08.914] [renderer1] [error] spawn cmd.exe ENOENT: Error: spawn cmd.exe ENOENT
	at Process.ChildProcess._handle.onexit (node:internal/child_process:282:19)
	at onErrorNT (node:internal/child_process:477:16)
	at processTicksAndRejections (node:internal/process/task_queues:83:21)
```

**Comment by adam8810** [comment]
@jan-dolejsi's PR has been merged and published. Please confirm this is fixed for you once you receive v1.3.0

**Comment by jan-dolejsi** [comment]
Works. Issue may be closed.
